packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -183,7 +182,7 @@ const main = async () => {
     const appSSRjs = fse.pathExistsSync(appSSRpath)
     const overrideSSRpath = p.join(
         process.cwd(),
-        projectPkg?.mobify?.overridesDir ?? '',
+        projectPkg?.ccExtensibility?.overridesDir ?? '',
```

Comment:
What does the "cc" stand for in "ccExtensibility"?

I know we have the discussion about there the extensibility key would live, package.json vs the project config. Can you remind me where what the technical reason why I couldn't be in the config?

Follow up,  is the `getConfig` util function extensibility aware? E.g. can configs be overridden?

Reply:
All of that is intentionally bypassed because for incoming MRT (Steel Arc) work, everything needs to be observable with `package.json` as the only reliable source of truth. Also, we agreed that those the existing `config` are categorically different from a "webpack build-level feature" which this feature is

Reply:
"cc" = "Commerce Cloud" (just to avoid namespace collision with some random other npm package)